design/one-pager-ignore-changes.md

```diff
@@ -0,0 +1,157 @@
+# Ignore field changes
+
+* Owners: Lovro Sviben (@lsviben)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Background
+
+When Crossplane manages a resource, it reconciles all of the parameters under `spec.forProvider` to the external providers both 
+during the creation and subsequent updates to the managed resource. But in some cases, a parameter of the external resource
+could change due it being managed outside of Crossplane. In the current state, if that happens, Crossplane will try to 
+"fix" the situation, and update the parameter back to the one it has in the `spec.forProvider` as it is designed to be
+the only source of truth.
+
+An example for this case is [AutoScalingGroup DesireCapacity], where when using an autoscaler, the parameter gets changed 
+externally. In this case, it would make sense to make Crossplane ignore this parameter during updates. For example Terraform has
+[ignore_changes].
```

Comment:
```suggestion
An example of this case is [AutoScalingGroup DesireCapacity], where when using an autoscaler, the parameter gets changed 
externally. In this case, it would make sense to have Crossplane ignore this parameter during updates.
Terraform addresses this use case with its [ignore_changes] functionality.
```